Formula/gtk-gnutella.rb

```diff
@@ -1,9 +1,10 @@
 class GtkGnutella < Formula
   desc "Share files in a peer-to-peer (P2P) network"
   homepage "https://gtk-gnutella.sourceforge.io"
-  url "https://downloads.sourceforge.net/project/gtk-gnutella/gtk-gnutella/1.2.0/gtk-gnutella-1.2.0.tar.xz"
-  sha256 "9608f28706f75423ac6b0d8f260506f1cf0f345a6f04de7cac1232b2504d94c9"
```

Comment:
why we want to use github tarball when their sha are different?

maybe a good thing to ask with the upstream repo. (Ideally, we'd like to use the end artifact rather than source code, sometimes could be source code layout issue)

Reply:
Okay, I'll wait a day or two and re-try and see if SF will work then